lumo-ui/plugin/src/main/java/com/nomanr/lumo/plugin/configs/ConfigurationValidator.kt

```diff
@@ -18,6 +18,7 @@ class ConfigurationValidator(private val project: Project, private val logger: L
             .replace("\\", ".") // Replace Windows-style backslashes
             .replace("/", ".")  // Replace Unix-style forward slashes
             .replace("//", ".") // Handle double forward slashes (if any)
+            .removeSuffix("/").removeSuffix("\\") // Remove last slash
```

Comment:
```suggestion
            .removeSuffix("/").removeSuffix("\\") // Remove the last slash or backslash
            .replace("\\", ".") // Replace Windows-style backslashes
            .replace("/", ".")  // Replace Unix-style forward slashes
            .replace("//", ".") // Handle double forward slashes (if any)
```

This should go at the top since we are replacing slashes with dots.